### PR TITLE
Edited Mercurial Support

### PR DESCRIPTION
**Pull request by zardra**

### CODE REVIEW

**Comment by facebook-github-bot-9** [comment]
By analyzing the blame information on this pull request, we identified @JoelMarcey and @matthewjohnston4 to be potential reviewers.

File: docs/_docs/features/hg.md [hunk 1]
```diff
@@ -19,19 +19,18 @@ integration with support for [Mercurial](https://www.mercurial-scm.org/).
...
 ![](/static/images/docs/feature-hg-diff-view-access.png)
 
-The Diff View is opened up in a new tab in the editor area. In the bottom pane are the files that
-have been modified, along with the Mercurial commit on which this diff is based. Click on the file
+The Diff View opens in a new tab in the [Editing Area](/docs/editor/basics/#editing). In the bottom pane are the files that
+have been modified, along with the Mercurial commit on which this diff is based. Click on a file
 in the bottom pane to see the changes for that file.
 
-In the left pane is what is currently committed for that file. In the right pane is what you have
+The left pane shows is what is currently committed for that file while the right pane displays what you have
```
Comment (JoelMarcey) [inline docs/_docs/features/hg.md]: remove the "is" after "shows"

comma after "file", I think .... "file, while..."

Comment (zardra) [inline docs/_docs/features/hg.md]: Fixed

**Comment by facebook-github-bot-9** [comment]
@zardra updated the pull request - [view changes](https://github.com/facebook/nuclide/pull/738/files/5296e8763dba0aed168b1ee9d73d3c7ab6ac6851..d942f649b61143626a1dd90559d4595fd17e6e10)

**Comment by facebook-github-bot-9** [comment]
Thanks for importing. If you are a Facebook employee, you can view this diff [on Phabricator](https://phabricator.intern.facebook.com/D3924980).

**Pull request closed without merging**